### PR TITLE
CEDS-2110 - tidy up code following review

### PR DESCRIPTION
**Pull request by SteveSugden**
These are some of the changes Maciej suggested after I merged in this feature.

### CODE REVIEW

File: app/models/viewmodels/ilequery/IleQueryConverter.scala [hunk 1]
```diff
@@ -41,6 +43,9 @@ class IleCodeMapper @Inject()(decoder: Decoder) {
   def statusOfEntryMucr(code: String)(implicit messages: Messages): Content =
     decoder.mucrSoe(code).map(soeCode => HtmlContent(htmlString(soeCode))).getOrElse(Empty)
 
-  def statusOfEntryCombined(code: String)(implicit messages: Messages): Content =
-    decoder.soeCombined(code).map(soeCode => HtmlContent(htmlString(soeCode))).getOrElse(Empty)
+  def statusOfEntryAll(code: String)(implicit messages: Messages): Content =
+    decoder.allSoe(code).map(soeCode => HtmlContent(htmlString(soeCode))).getOrElse(Empty)
+
+  def transport(transport: Transport): Content =
+    Text((transport.transportId.toSeq ++ transport.nationality.map(countryName).toSeq).mkString(", "))
```
Comment (MaciejRewera) [inline app/models/viewmodels/ilequery/IleQueryConverter.scala]: I've just checked and you don't need `.toSeq` calls. Can do just `Text((transport.transportId ++ transport.nationality.map(countryName)).mkString(", "))`

File: app/models/viewmodels/ilequery/IleQueryConverter.scala [hunk 2]
```diff
@@ -14,15 +14,17 @@
...
 import javax.inject.{Inject, Singleton}
+import models.notifications.queries.Transport
 import models.viewmodels.decoder.{CodeWithMessageKey, Decoder, ROECode}
 import play.api.i18n.Messages
+import services.Countries.countryName
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Empty, HtmlContent, Text}
 
 @Singleton
-class IleCodeMapper @Inject()(decoder: Decoder) {
+class IleQueryConverter @Inject()(decoder: Decoder) {
```
Comment (MaciejRewera) [inline app/models/viewmodels/ilequery/IleQueryConverter.scala]: `IleQueryCodeConverter` ? At first I thought it is converting the whole IleQuery

File: app/views/ile_query_ducr_response.scala.html [hunk 3]
```diff
@@ -46,48 +47,8 @@
...
-                        ),
-                        value = Value(
-                            content = info.entryStatus.flatMap(_.ics).map(ileCodeMapper.inputCustomsStatus(_)).getOrElse(Empty)
-                        )
-                    )
-                ).filterNot(row => row.key.content == Text(messages("ileQueryResponse.details.transport")) && row.value.content == Empty),
-                classes = "govuk-!-margin-bottom-9"
-            ))
-        </div>
+        @summarySection(info)
```
Comment (MaciejRewera) [inline app/views/ile_query_ducr_response.scala.html]: 👍

File: app/views/ile_query_mucr_response.scala.html [hunk 4]
```diff
@@ -57,40 +58,7 @@ <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">@messages("ileQueryResponse
...
-                        ),
-                        value = Value(
-                            content = info.transport.map(transportMapper.transportHtml).getOrElse(Empty)
-                        )
-                    )
-                ).filterNot(row => row.key.content == Text(messages("ileQueryResponse.details.transport")) && row.value.content == Empty),
-                classes = "govuk-!-margin-bottom-9"
-            ))
-        </div>
+        @summarySection(info)
```
Comment (MaciejRewera) [inline app/views/ile_query_mucr_response.scala.html]: 👍

File: app/models/viewmodels/ilequery/IleQueryConverter.scala (hunk 2 above)
Comment (SteveSugden) [reply app/models/viewmodels/ilequery/IleQueryConverter.scala]: Renaming

File: app/models/viewmodels/ilequery/IleQueryConverter.scala (hunk 1 above)
Comment (SteveSugden) [reply app/models/viewmodels/ilequery/IleQueryConverter.scala]: You are correct.  Changed and created test for it

**Pull request merged**